web/e2e/README.md

```diff
@@ -0,0 +1,90 @@
+# Playwright E2E Testing Project
+
+## Project Overview
+
+This project is an end-to-end (E2E) testing suite built using [Playwright](https://playwright.dev/). It follows the Page Object Model (POM) design pattern to organize tests for various components of the `visualizer` module, including `dashboard`, `projectSetting`, and `scene`.
+
+## Project Structure
+
+web
+├── e2e
+│ ├── pom
+│ │ ├── visualizer
+│ │ │ ├── dashboard
+│ │ │ │ ├── index.ts
+│ │ │ │ └── ProjectsPage.ts
+│ │ │ ├── projectSetting
+│ │ │ │ ├── generalPage.ts
+│ │ │ │ └── index.ts
+│ │ │ ├── editor
+│ │ │ │ ├── index.ts
+│ │ │ │ └── MapPage.ts
+│ │ └── index.ts
+│ ├── utils
+│ │ ├── .auth
+│ │ ├── config.ts //old setting
+│ │ ├── index.ts //old setting
+│ │ ├── login.ts //old setting
+│ │ ├── setup.ts //old setting
+│ │ └── auth.setup.ts
+│ ├── dashboard.spec.ts //old testing
+│ └── test.spec.ts //testing file
+└── README.md
+
+## Prerequisites
+
+- [Node.js](https://nodejs.org/) >= 14.x
+- [npm](https://www.npmjs.com/) or [Yarn](https://yarnpkg.com/)
+
+## Setup
+
+### Install Dependencies
+
+To install the required packages, run the following commands:
+
+```bash
+cd web
+yarn install
+npx playwright install //if needed
+```
+
+To set required .env for frontend. Recording following variable.
+
+```bash
+# Local backend with Auth0 OSS tenant
+REEARTH_WEB_E2E_BASEURL=http://localhost:3000
+REEARTH_WEB_E2E_ACCOUNT // your visualizer accounnt
+REEARTH_WEB_E2E_ACCOUNT_PASSWORD // your visualizer password
+```
```

Comment:
Sorry, we might need to add these to `.env.example` file which acts as a template for what variables are required to run visualizer.

Reply:
Thank you for your advice. I already add these three variables to `.env.example`. Please check.
https://github.com/reearth/reearth-visualizer/pull/1140/commits/e9160e5ece3adad59c381582a337f24662533aae

Reply:
Thanks. Looks good.